### PR TITLE
Set up NPM workspaces

### PR DESCRIPTION
**Pull request by 7opf**
## Description

Sets up a monorepo structure for the repo using npm workspaces. Adds a "common" package with only "TextField" as an exported component for now to test it works. READMEs and workflows are updated to reflect use of npm workspaces.

fyi @obany has given this a review.

## Issues

* Closes https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/issues/23
* Related https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/issues/85

## Dependencies

* https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/pull/111/

### CODE REVIEW

File: docs/tutorials/101/zebra-iota-edge-sdk-101-tutorial.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ adb logcat | grep -F "`adb shell ps | grep org.iota.zebra | tr -s [:space:] ' '
 
 ### Getting started
 
-Under the[ identity-enabler](https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK) folder, there is one subfolder for each of the reference applications. For each application the `src` directory contains the application source code and static web assets. Additionally, each application has a `config.ts` file with settings and contains an Android application in the `android` directory.
+The repository is set up as a monorepo using npm workspaces. Under the [identity-enabler](https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK) folder, there is one subfolder for each of the reference applications. For each application the `src` directory contains the application source code and static web assets. Additionally, each application has a `config.ts` file with settings and contains an Android application in the `android` directory. The `packages` directory contains libraries shared by the applications, currently only the `common` package.
```
Comment (jmcanterafonseca-iota) [inline docs/tutorials/101/zebra-iota-edge-sdk-101-tutorial.md:140]: a link to the npm workspaces introduction page would help

File: packages/common/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Zebra IOTA Edge SDK Common Library
+
+A base for building shareable Svelte components. Clone it with [degit](https://github.com/Rich-Harris/degit):
```
Comment (jmcanterafonseca-iota) [inline packages/common/README.md:3]: why degit is needed?

**Review by jmcanterafonseca-iota — COMMENTED** [review]
LGTM with the minor questions responded

File: packages/common/README.md (hunk 2 above)
Comment (7opf) [reply packages/common/README.md:3]: This is copy pasta. I will remove.

**Review by jmcanterafonseca-iota — APPROVED** [review]

**Pull request merged**